labs-ios-starter/Surveys/View Controllers/TopicViewController.swift

```diff
@@ -76,15 +80,26 @@ class TopicViewController: UIViewController {
         }
     }
 
-    // MARK: - Navigation -
-
-
+    // MARK: - Pull to Refresh
+    /// Configures the collectionView's refreshControl
+    private func configureRefreshControl() {
+        refreshControl.addTarget(self, action: #selector(refreshControlHandler), for: .valueChanged)
+        topicsCollectionView.alwaysBounceVertical = true
```

Comment:
have you tried disabling bounce to help with the refresh issue you discussed? It could be a timing issue that has to do with the animation. If disabling fixes it, we might be able to find a workaround with a completion-handler or something